### DISCUSSION TITLE
Is it posible to block the canvas resize?

### DISCUSSION THREAD

**Discussion by jiagaya** [rustdesk/rustdesk#8571] [Feature Request]
I use rustdesk to control my pc from my android tablet. Out of intuition I try to zoom with two fingers. What instead of zooming in my application, the canvas of the duplicate screen changes, so I have to go back to settings and change the screen settings so that it adapts to my tablet. Is it possible to lock canvas movement/scaling?
Thank you

**Comment by yuyehk** [comment]
+1
